client/pipeline_parameter.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+// "parameterConfig": [
```

Comment:
do we want to keep this?

Reply:
I was just going to remove it in the next commit.